src/pybind/rbd/rbd.pyx

```diff
@@ -2964,6 +2964,36 @@ cdef class Group(object):
         if ret != 0:
             raise make_ex(ret, 'error rolling back group to snapshot', group_errno_to_exception)
 
+    def mirror_group_create_snapshot(self, flags=0):
+        """
+        Create mirror group snapshot.
+
+        :param flags: create snapshot flags
+        :type flags: int
+        :returns: str - group snapshot ID
+        """
+        cdef:
+            uint32_t _flags = flags
+            char *snap_id = NULL
+            size_t max_snap_id_size = 15
```

Comment:
Can this be defined somewhere instead of being hardcoded?

Reply:
I've made the changes here, https://github.com/ceph/ceph/pull/60627/commits/89321c04b678b2006f9bdfc13ff3432210df4c52

Also note that rbd_mirror_group_create_snapshot() C API is the one that correctly sets the max_snap_id_size. The cython APIs are designed to resize the output result based on the C API results